ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -78,3 +78,9 @@ def hstack(tup, dtype=None):
 @to_ivy_arrays_and_back
 def eye(N, M=None, k=0, dtype=None):
     return ivy.eye(N, M, k=k, dtype=dtype)
+
+
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+def triu(m, k=0, dtype=None):
```

Comment:
I think the original function does not have a `dtype` argument, does it?

Reply:
No it doesn't but without the `dtype` argument the function doesn't work, and the some of the other functions in the file doesn't require dtype, but its still passed.

Reply:
What do you mean it doesn't work? You are not using this argument anywhere in the function.

Reply:
Yes, I had an issue(it doesn't work) because i added decorator `@handle_numpy_dtype`. Now I removed it and everything works fine.